osquery/filesystem/filesystem.cpp

```diff
@@ -363,7 +408,7 @@ Status resolveFilePattern(std::vector<std::string> components,
 
 Status resolveFilePattern(const boost::filesystem::path& fs_path,
                           std::vector<std::string>& results) {
-  return resolveFilePattern(split(fs_path.string(), "/"), results, 0, 0);
+  return resolveFilePattern(split(fs_path.string(), "/"), results);
```

Comment:
Line 322 takes the resolved components and creates the absolute path out of them. The components are the parts of an absolute filepath split at /